shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -166,7 +166,10 @@
 // |PlatformView|
 void PlatformViewIOS::UpdateSemantics(flutter::SemanticsNodeUpdates update,
                                       flutter::CustomAccessibilityActionUpdates actions) {
-  FML_DCHECK(owner_controller_);
+  if (!owner_controller_) {
```

Comment:
It's hard to know that this change is valid without a unit test, please add one.

Reply:
postNotificationName:object: cannot take a nil object reference.

Reply:
Can't it?  The swift declaration has the second parameter being a nullable type.

Reply:
I believe that just throws an ObjC exception. So, either don't pass null or add a try-catch block.

Reply:
Passing in a nil for object isn't an ObjC exception, or a crash.  I think passing nil actually broadcasts to all people subscribed to the notification, similar to calling `postNotification:`.  The crash has to be downstream in the handler.  If that's the case there could be a break introduced by this.  Since posting a notification with a nil object might be a valid flow if the handler accounts for the possibility of a nil object.

We never did get a stack trace from the crash they were fixing.